agimus_controller_ros/controller_base.py

```diff
@@ -170,33 +240,65 @@ def solve(self):
         x0 = np.concatenate(
             [sensor_msg.joint_state.position, sensor_msg.joint_state.velocity]
         )
+        self.target_translation_object_to_effector = None
+        self.in_world_M_effector = None
         self.fill_buffer()
         if self.traj_buffer.get_size(self.point_attributes) > 0:
             point = self.traj_buffer.get_points(1, self.point_attributes)[0]
             self.last_point = point
+            new_x_ref = point.get_x_as_q_v()
+            self.mpc_data["hpp_trajectory"].append(new_x_ref)
+
         else:
             point = self.last_point
-        new_x_ref = point.get_x_as_q_v()
```

Comment:
This is perfectly fine so why put 2 identical line in the if and else?

Reply:
same answer